matplotlibrc.template

```diff
@@ -312,7 +312,10 @@ backend      : %(backend)s
 #axes.unicode_minus  : True    # use unicode for the minus symbol
                                # rather than hyphen.  See
                                # http://en.wikipedia.org/wiki/Plus_and_minus_signs#Character_codes
-#axes.prop_cycle    : cycler('color', 'bgrcmyk')
+#axes.prop_cycle    : cycler('color',
+#                            ['#1f77b4', '#ff7f0e', '#2ca02c', '#d62728',
+                              '#9467bd', '#8c564b', '#e377c2', '#7f7f7f',
+                              '#bcbd22', '#17becf'])
```

Comment:
rc files do not use the hash symbol for the hex colorspec. leave off the hash. You are also missing the comment symbol at the beginning of two lines.